### ISSUE TITLE
FEATURE - Cascade Delete `MaintenanceEvent`s When Vehicles are Deleted

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#160]
## Describe the feature you want the app to have that it currently doesn't
* When a `Vehicle` is deleted on the `SettingsView` the `MaintenanceEvent`s for that vehicle also need to be deleted

## Describe alternatives you've considered
* N/A

## Additional context
* This issue can not be worked on until #141 is done

**Comment by Drag0ndust** [comment]
@mikaelacaron May I take this issue, when it is unblocked?

**Comment by mikaelacaron** [comment]
Yup! #163 is up, I'll be looking at it tonight

**Comment by Drag0ndust** [comment]
is this still on hold?
The PR, mentioned above, has been merged as I can see

**Comment by mikaelacaron** [comment]
Actually hold a little longer, but DM me on social media (if you want to work on this), there's actually an issue with the current solution

I was going to fix this tonight, if I have time

**Comment by Drag0ndust** [comment]
Too late :)

Already worked on it and the PR is almost ready

**Comment by TratonGossink** [comment]
Hi Mikael, 
Is this issue still available to be worked on?

**Comment by mikaelacaron** [comment]
@TratonGossink Can you actually hold on this one right now. I'm about to change how vehicles are saved with maintenance events, which will affect how this issue is solved

**Comment by mikaelacaron** [comment]
Closing this because it's on hold for right now
